libraries/provider_dnsimple_record.rb

```diff
@@ -68,9 +68,9 @@ def load_current_resource
 
       def create_record
         converge_by("create record #{new_resource.record_name} for domain #{new_resource.domain}") do
-          dnsimple_client.zones.create_record(
-            dnsimple_client_account_id, new_resource.domain, **record_options
-          )
+          dnsimple_client.zones.create_zone_record(dnsimple_client_account_id,
+                                                   new_resource.domain,
+                                                   **record_options)
```

Comment:
We should maintain parens on it's own line when using multiline methods for readability
```suggestion
                                                   **record_options
                                                   )
```